.github/workflows/netlify_cms_config_validation.yml

```diff
@@ -0,0 +1,35 @@
+name: Netlify CMS config validation
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v1
+    - name: Parse config file
+      id: config
+      run: |
+        sudo snap install yq
+        echo "##[set-output name=result;]$(yq r ${{ env.CONFIG_PATH }} --tojson)"
```

Comment:
[`yq`](https://github.com/mikefarah/yq) is a command-line YAML parser. This installs `yq`, parses our Netlify CMS configuration file, converts in to JSON, and sets it to an output that can be [accessed by future steps](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#steps-context).